src/components/TeamCard/TeamCard.scss

```diff
@@ -0,0 +1,70 @@
+@use "~stylesheets/typography";
+
+.Card {
+  animation: fadeIn 1s;
+  width: 292px;
+  height: 392px;
+  margin-left: 10px;
+  margin-right: 10px;
+  border-radius: 27px;
+  padding-bottom: 10px;
+  background:  
+    linear-gradient(#34486B, #34486B) padding-box,
+    linear-gradient(to right, #BD9BF3, #C0CAFC, #80B0F8) border-box;
```

Comment:
I'll say let's wait on making this change until after everything is merged in but we should create variables for these.